src/App.tsx

```diff
@@ -1,29 +1,25 @@
 import "./App.css";
-import React, { useState } from "react";
+import { useState } from "react";
 import ButtonWithInput from "./components/ButtonWithInput.tsx";
 import Button from "./components/Button.tsx";
 import Count from "./components/Count.tsx";
-import {nanoid} from "nanoid";
+import ListUser from "./components/ListUser.tsx";
 
 const list = [
```

Comment:
```suggestion
const userList: User[] = [
```

иначе кто-нибудь, где-нибудь в коде добавит в массив объект без поля name или id и твой код отработает некорректно